policies/cla.html

```diff
@@ -46,8 +46,10 @@
 	  <p>
 	  In practice, it is required that the author (in the git commit
 	  message) and all approving team members (in the pull request thread)
-	  agree that a change is trivial. The reviewers will normally post
-	  a statement to the effect of "I agree that it is a trivial change."
+	  agree that a change is trivial. The author has to add "CLA: trivial"
```

Comment:
Given the confusion that just happened, perhaps add "(note that there is a space after the colon)"